src/services/at-sce-api.service.js

```diff
@@ -58,8 +77,18 @@ AT_SCE_SERVICE.createUser = (data) => {
   });
 };
 
+// Operation; Create USER - POST /api/v1/users
+AT_SCE_SERVICE.createUser = () => {
```

Comment:
this function already exists https://github.com/at-internship/at-sce-ui/blob/develop/src/services/at-sce-api.service.js#L48:L59